# singlecell-sporocysts
This repository contains the scripts used for the analysis of single cell RNA-seq data from S. mansoni sporocysts.

1. ###QC_sporos.R: QC of sporocyst samples. This script was written and run by cdsoria (https://github.com/cdsoria)
2. ###Analysis_sporos.R: Inital analysis of sporocyst samples. This script was written and run by cdsoria (https://github.com/cdsoria)
3. ###Transfer_annotation.R: Transfer of somule annotations to sporocyst samples. This script was written and run by cdsoria (https://github.com/cdsoria)
4. ###go_terms_muscle.Rmd: Muscle GO term comparison across miracidia, sporocysts, somules, and adults
5. ###go_terms_neurons_ll.Rmd: Neuron GO term comparison across miracidia, sporocysts, somules, and adults
6. ###go_terms_parenchyma.Rmd: Parenchyma GO term comparison arcross miracidia, sporocysts, somules, and adults
7. ###go_terms_stem.Rmd: Stem GO term comparison across miracidia, sporocysts, somules, and adults
8. ###go_terms_tegument.Rmd: Tegument GO term comparison across miracidia, sporocysts, somules, and adults
9. ###marker_genes_across_muscle.Rmd: Muscle marker gene comparison across miracidia, sporocysts, somules, and adults
10. ###marker_genes_across_neurons.Rmd: Neuron marker gene comparison across miracidia, sporocysts, somules, and adults
11. ###marker_genes_across_parenchyma.Rmd: Parenchyma marker gene comparison across miracidia, sporocysts, somules, and adults
12. ###marker_genes_across_stem.Rmd: Stem marker gene comparison across miracidia, sporocysts, somules, and adults
13. ###marker_genes_across_tegument.Rmd: Tegument marker gene comparison across miracidia, sporocysts, somules, and adults
14. ###sam_on_sporos-3perc-stem.ipynb: SAM analysis on stem cells
15. ###sam_on_sporos-3perc.ipynb: SAM analysis on all cells
16. ###teg1_v_teg2_sporo.Rmd: DE analysis on teg 1 v teg 2 clusters
17. ###sporo3perc_analysis_paper.Rmd: continued analysis of sporocysts
18. ###top_markers_3percSporos.Rmd: pulling top markers from each cluster
19. ###threeperc_figs.Rmd: extra figures e.g. top marker dotplots

##Mapping: 10X Chromium single-cell RNAseq reads were mapped using CellRanger. The code for CellRanger is accessible from the 10X website. 
